tests/unit-tests/jobs/models/jobExperimentsHandler-test.js

```diff
@@ -49,70 +53,87 @@ describe('Job experiments handler tests', function () {
         sandbox.restore();
     });
 
-    it('set chaos experiments with 2 experiments', async () => {
-        const firstExperiment = generateExperiment();
-        const secondExperiment = generateExperiment();
-        const firstExperimentName = firstExperiment.kubeObject.metadata.name;
-        experimentsManagerInsertStub.resolves();
-        experimentsManagerGetStub.resolves([firstExperiment, secondExperiment]);
-        experimentsManagerRunJobStub.resolves();
-        const jobExperiments = [
-            {
-                experiment_id: firstExperiment.id,
-                start_after: 1000
-            },
-            {
-                experiment_id: secondExperiment.id,
-                start_after: 2000
-            }
-        ];
-        const jobId = uuid();
-        clock = sinon.useFakeTimers();
-        clock.tick(1000);
-        await jobExperimentHandler.setChaosExperimentsIfExist(jobId, jobExperiments);
-        clock.tick(3000);
-        (experimentsManagerRunJobStub.callCount).should.eql(2);
-        experimentsManagerRunJobStub.args[0][0].metadata.name.should.eql(`${firstExperimentName}-${experimentsManagerRunJobStub.args[0][2]}`);
-        experimentsManagerGetStub.callCount.should.eql(1);
-        experimentsManagerInsertStub.callCount.should.eql(2);
-        experimentsManagerInsertStub.args[0][1].should.eql(jobId);
-        experimentsManagerInsertStub.args[0][2].should.eql(firstExperiment.id);
-        (experimentsManagerInsertStub.args[0][4] - experimentsManagerInsertStub.args[0][3]).should.eql(60000);
-        experimentsManagerInsertStub.args[1][1].should.eql(jobId);
-        experimentsManagerInsertStub.args[1][2].should.eql(secondExperiment.id);
-        (experimentsManagerInsertStub.args[1][3] - experimentsManagerInsertStub.args[0][3]).should.eql(1000);
-    });
+    describe('Set experiments if exist tests', function () {
+        it('set chaos experiments with 2 experiments', async () => {
```

Comment:
it('set chaos experiments with 2 experiments', async () => { this is not very clear
you meant set job with 2 experiments

Reply:
fixed